helm-projectile.el

```diff
@@ -209,8 +209,10 @@ It is there because Helm requires it."
 (defcustom helm-source-projectile-projects-actions
   (helm-make-actions
    "Switch to project" (lambda (project)
-                         (let ((projectile-completion-system 'helm))
-                           (projectile-switch-project-by-name project)))
+                         (let ((project-path (expand-file-name project)))
```

Comment:
I don't get what this aims to achieve - I'm assuming if you just need to run `helm-project` in the context of a project you can let-bind `default-directory`.

Reply:
Haha OK that's easier :) I was thinking this was overkill over in the issue…

I will force push to keep a single commit in the pull request.